docs/docs/test-runner/overview.md

```diff
@@ -8,13 +8,14 @@ eleventyNavigation:
 
 Test runner for web applications.
 
-👉&nbsp;&nbsp; Headless browsers with puppeteer, playwright, or selenium. <br>
+👉&nbsp;&nbsp; Headless browsers with [Puppeteer](https://modern-web.dev/docs/test-runner/browsers/puppeteer/), [Playwright](https://modern-web.dev/docs/test-runner/browsers/playwright/), or [Selenium](https://modern-web.dev/docs/test-runner/browsers/selenium/). <br>
 🚧&nbsp;&nbsp; Reports logs, 404s, and errors from the browser. <br>
-📦&nbsp;&nbsp; Supports native es modules.<br>
-🔧&nbsp;&nbsp; Runs tests in parallel and isolation.<br>
+🔍&nbsp;&nbsp; Debug opens a real browser window with devtools.<br>
+🔧&nbsp;&nbsp; Exposes browser properties like viewport size and dark mode.<br>
+⏱&nbsp;&nbsp;Runs tests in parallel and isolation.<br>
 👀&nbsp;&nbsp; Interactive watch mode.<br>
-🏃&nbsp;&nbsp; Reruns only changed tests.<br>
-🚀&nbsp;&nbsp; Powered by [esbuild](../../dev-server/plugins/esbuild.md) and [rollup plugins](../../dev-server/plugins/rollup.md)
+🏃&nbsp;&nbsp; Fast development by rerunning only changed tests.<br>
+🚀&nbsp;&nbsp; Powered by [esbuild](https://modern-web.dev/docs/dev-server/esbuild.md) and [rollup plugins](https://modern-web.dev/docs/dev-server/rollup.md)
```

Comment:
why use full urls (incl. `https://modern-web.dev`)?

Reply:
this was a copy paste from the blog post which had it like that

Reply:
fixed everywhere